capabilities/index/index_test.go

```diff
@@ -0,0 +1,37 @@
+package index_test
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/ipfs/go-cid"
+	"github.com/ipld/go-ipld-prime/codec/dagjson"
+	cidlink "github.com/ipld/go-ipld-prime/linking/cid"
+	basicnode "github.com/ipld/go-ipld-prime/node/basic"
+	"github.com/storacha/go-libstoracha/capabilities/index"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRoundTripAddCaveats(t *testing.T) {
+	testCid, err := cid.Parse("bafybeiduiecxoeiqs3gyc6r7v3lymmhserldnpw62qjnhmqsulqjxjmtzi")
```

Comment:
I think we can use https://github.com/storacha/go-libstoracha/blob/main/capabilities/internal/testutil/gen.go#L17 `RandomCID` here, which returns an `ipld.Link`